scripts/pysa_fuzzer/hinder_static_analysis.py

```diff
@@ -0,0 +1,134 @@
+# hindering_static_analysis.py
+
+# 1. Indirect Function Calls
+def example_indirect_function_calls():
+    def source():
+        return input()
+
+    def sink(x):
+        print(x)
+
+    def indirect_call(f, arg):
+        return f(arg)
+
+    data = source()
+    indirect_call(sink, data)
+
+# 2. Use of exec and eval
+def example_exec_eval():
+    source_code = "a = input()"
+    sink_code = "print(a)"
+    exec(source_code)
+    exec(sink_code)

```

Comment:
It is a known false-negative, so there is no point generating this kind of code. We already know about the problem.